modules/pages/client/controllers/home.client.controller.js

```diff
@@ -36,6 +37,15 @@ function HomeController($stateParams, $window, TribesService, TribeService) {
   // Exposed to the view
   vm.boardHeight = $window.innerWidth <= 480 && $window.innerHeight < 700 ? 400 : $window.innerHeight - headerHeight + 14;
 
+  vm.statisticsData = statisticsData;
```

Comment:
No need to expose this if you're already exposing `vm.usersNo`, you can just default the value of `usersNo` to `40,000+` and use it directly at the template without using `ng-if`.